src/javaRosa.js

```diff
@@ -1976,6 +1976,7 @@ define([
             );
 
             ret = ret.concat([
+                'appearance',
```

Comment:
Yeah. I think that was in my other addition to javaRosa. Changed it over to core. Should be no more changes to javaRosa here